### PR TITLE
Dark Mode implementation

### PR DESCRIPTION
**Pull request by akrol95**

### CODE REVIEW

File: BMM.Core/Models/PO/BasePO.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+using MvvmCross.ViewModels;
+
+namespace BMM.Core.Models.PO
+{
+    public abstract class BasePO : MvxNotifyPropertyChanged
```
Comment (kkuepper) [inline BMM.Core/Models/PO/BasePO.cs:5]: what does PO stand for?

File: BMM.UI.Android/Application/NewMediaPlayer/Controller/AndroidMediaPlayer.cs [hunk 2]
```diff
@@ -147,7 +153,7 @@ public async Task Play(IList<IMediaTrack> mediaTracks, IMediaTrack currentTrack,
             }
         }
 
-        public async Task RecoverQueue(IList<IMediaTrack> mediaTracks, IMediaTrack currentTrack, string playbackOrigin, long startTimeInMs = 0)
+        public async Task RecoverQueue(IList<IMediaTrack> mediaTracks, IMediaTrack currentTrack, string playbackOrigin = "", long startTimeInMs = 0)
```
Comment (kkuepper) [inline BMM.UI.Android/Application/NewMediaPlayer/Controller/AndroidMediaPlayer.cs:156]: why not null?

File: BMM.Core/Models/PO/BasePO.cs (hunk 1 above)
Comment (akrol95) [reply BMM.Core/Models/PO/BasePO.cs:5]: Presentation Object, it is basically model of object which interacts with view and it not View Model, thats why it implements INotifyPropertyChanged. It is used e.g. for cells or for custom views.

File: BMM.UI.Android/Application/NewMediaPlayer/Controller/AndroidMediaPlayer.cs (hunk 2 above)
Comment (kkuepper) [reply BMM.UI.Android/Application/NewMediaPlayer/Controller/AndroidMediaPlayer.cs:156]: investigate removing JsonIgnore and simplifying the code around playback origin

File: BMM.Core/Models/PO/BasePO.cs (hunk 1 above)
Comment (kkuepper) [reply BMM.Core/Models/PO/BasePO.cs:5]: please add a summary comment to BasePO explaining it.

**Review by kkuepper — APPROVED** [review]

**Pull request merged**